src/SFA.DAS.Forecasting.Web/Filters/EmployerRecruitClaims.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.Forecasting.Web.Filters
+{
+    public static class EmployerRecruitClaims
```

Comment:
This has been copied and pasted and not renamed.

Reply:
👍